presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -144,7 +191,6 @@ private RuleApplication applyRule()
   {
       SymbolAllocator symbolAllocator = new SymbolAllocator(symbols);
       Memo memo = new Memo(idAllocator, plan);
-      Lookup lookup = Lookup.from(memo::resolve, statsCalculator, costCalculator);
```

Comment:
now you construct new `memo` but you do not make `lookup` out of it. I don't understand